x/ibc/04-channel/client/cli/tx.go

```diff
@@ -52,7 +53,7 @@ func NewChannelOpenInitCmd() *cobra.Command {
 		},
 	}
 	cmd.Flags().Bool(FlagOrdered, true, "Pass flag for opening ordered channels")
-	cmd.Flags().String(FlagIBCVersion, types.DefaultChannelVersion, "supported IBC version")
```

Comment:
this maybe should just be an argument, since the default depends on what application you are creating a channel for, follow up pr since there is a todo for fixing counterparty version as well